backend/app/db/collectionWatcher.js

```diff
@@ -2,30 +2,30 @@ const MongoClient = require('mongodb').MongoClient;
 const EventEmitter = require('events');
 
 class CollectionWatcher extends EventEmitter {
-  constructor() {
-    super();
-    this.collections = [];
-  }
+	constructor() {
+		super();
+		this.collections = [];
+	}
 
-  async init(clientUrl) {
-    this.client = await MongoClient.connect(clientUrl);
-  }
-  subscribeToCollection(collection) {
-    const mongoCollection = this.client.db('rockandrolla').collection(collection);
-    const collectionStream = mongoCollection.watch();
+	async init(clientUrl) {
+		this.client = await MongoClient.connect(clientUrl);
+	}
+	subscribeToCollection(collection) {
+		const mongoCollection = this.client.db('rockandrolla').collection(collection);
+		const collectionStream = mongoCollection.watch();
     
-    this.collections.push(
-      {
-       [collection]: collectionStream
-      });
+		this.collections.push(
+			{
+				[collection]: collectionStream
+			});
     
-    collectionStream.on('change', (change) => {
-      this.emit(`${collection}:change`, change);
-    });
-  }
+		collectionStream.on('change', (change) => {
+			this.emit(`${collection}:change`, change);
+		});
+	}
 
 }
 
 module.exports = {
-  CollectionWatcher
-}
\ No newline at end of file
+	CollectionWatcher
+};
\ No newline at end of file
```

Comment:
The code changes appear to be mostly formatting related, with indentation and white space adjustments. There do not seem to be any functional changes. 

As for potential improvements, one suggestion could be to add error handling to the `MongoClient.connect()` method call in the `init(clientUrl)` function. Additionally, it may be useful to consider adding a way to unsubscribe from collections if necessary, as currently collections are added to the `collections` array but there is no way to remove them.